pandas/core/series.py

```diff
@@ -2208,6 +2208,8 @@ def combine(self, other, func, fill_value=np.nan):
         func : function
             Function that takes two scalars as inputs and return a scalar
         fill_value : scalar value
+            The default specifies to use the appropriate NaN value for
+            the underlying dtype of the Series
```

Comment:
does this need a versionchanged?

Reply:
There should be no change in behaviour for normal Series I think, as the `na_value_for_dtype`will give NaN/NaT (which was the default before). It's only for extension arrays that it might give another value, depending on what the extension array defined its missing value to be.

Reply:
I agree with @jorisvandenbossche